coffee-shop/src/genericComponents/Header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
you need to add index file in the header component folder as we did in pages

Reply:
done